SourceryTests/Parsing/FileParser + VariableSpec.swift

```diff
@@ -115,6 +115,10 @@ class FileParserVariableSpec: QuickSpec {
                     expect(parse("var name: Int { didSet {} }")).to(equal(Variable(name: "name", typeName: TypeName("Int"), accessLevel: (read: .internal, write: .internal), isComputed: false)))
                 }
 
+                it("extracts lazy property correctly") {
```

Comment:
please add the test instead in a `FileParserAttributesSpec` spec, there is a context for variables, the test will also become shorter. The test fails because you don't provide `description` parameter, so it makes it as `@lazy`. You'll see similar test for `final` attribute in `FileParserAttributesSpec`

Reply:
Gonna jump into this!